### ISSUE TITLE
[ChannelTabs v1.2.0] Issue: Requests for Account claiming

### ISSUE THREAD

**Issue by PaxiTheAlchemist** [l0c4lh057/BetterDiscordStuff#53]
# Describe the bug

Upon Enabling or disabling the plugin from the Plugin Menu, you are redirected to the "My Account" tab of the Settings where Discord then requests for me to claim an account. No other plugins are enabled throughout this.

# To Reproduce

## Prior Requirements

- Have Better Discord installed
- Have on hand the ChannelTabs v1.2.0 js file in the plugins folder

## Steps to reproduce the behavior:

1. Upon starting a fresh Better Discord session, navigate to the Plugins tab within the settings
2. Navigate to the ChannelTabs v1.2.0 entry within the Plugins window
3. Enable or Disable the plugin

# Expected behavior

The plugin should simply enable or disable without Discord requesting for the user to claim an account

# Screenshots

![](https://i.imgur.com/ZULIqaC.gif)

# Information (please complete the following information):

 - OS: Windows
 - Plugin Version: v1.2.0
 - Discord version: Stable and Canary
 - Compact Mode enabled? no

**Comment by Strencher** [comment]
did you saw this?
![img](https://i.imgur.com/aqKJG6P.png)

**Comment by l0c4lh057** [comment]
I am aware as I stated in the repo channels on the servers. The problem is that those don't seem to be the exact steps to reproduce because that never happened to me. There will be an update in the future and I hope it will fix this issue but I can't tell if that actually is the case.

**Comment by PaxiTheAlchemist** [comment]
Interesting. What do you suspect the issue could be? or is it possibly something out of your control? When your plugin first released on the BD Repo, i don't recall having the issue when i first installed it. However, when i first discovered it, i honestly thought i may have contracted a virus on my client or something. This being especially so with Discord requiring account info in the claiming process. In response to this, I reinstalled Discord, which only lead to the same thing to happen upon re-enabling the plugin. 

also, my apologies @Strencher, I'm not in the support server and thus i didn't know that it was an already discovered issue. I just looked through the issues in this GitHub repository and saw that there was nothing relating to this issue present here. And also yes, i have figured out on my own that you can ignore the message but I also decided to report the issue for the reasons explained above.

**Comment by l0c4lh057** [comment]
Yeah no problem, I rather get something reported twice than never. I really have no idea what causes it, it is just my hope that what I have in mind will fix it, I don't know though

**Comment by azbcethananderson** [comment]
can we get tabs for threads? 
theres no option currently
